common/concept.proto

```diff
@@ -107,19 +107,19 @@ message Thing {
     bytes iid = 1;
     Type type = 2;
     Attribute.Value value = 3;
+    bool inferred = 4;
 
     message Req {
         bytes iid = 1;
         oneof req {
             // Thing method requests
             Thing.Delete.Req thing_delete_req = 100;
             Thing.GetType.Req thing_get_type_req = 101;
-            Thing.IsInferred.Req thing_is_inferred_req = 102;
-            Thing.GetHas.Req thing_get_has_req = 103;
-            Thing.SetHas.Req thing_set_has_req = 104;
-            Thing.UnsetHas.Req thing_unset_has_req = 105;
-            Thing.GetRelations.Req thing_get_relations_req = 106;
-            Thing.GetPlaying.Req thing_get_playing_req = 107;
+            Thing.GetHas.Req thing_get_has_req = 102;
+            Thing.SetHas.Req thing_set_has_req = 103;
+            Thing.UnsetHas.Req thing_unset_has_req = 104;
+            Thing.GetRelations.Req thing_get_relations_req = 105;
+            Thing.GetPlaying.Req thing_get_playing_req = 106;
```

Comment:
Note that moving these around breaks compatibility with older clients - maybe this is a practice we want to abolish going forward?